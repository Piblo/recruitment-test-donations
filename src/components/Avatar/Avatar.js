import styled from 'styled-components';

const Avatar = styled.img`
  border-radius: 50%;
  height: 100%;
`;

export default Avatar;
