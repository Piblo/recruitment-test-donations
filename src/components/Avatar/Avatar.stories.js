import React from 'react';
import { storiesOf } from '@storybook/react/dist/client/preview';
import Avatar from './Avatar';

storiesOf('Avatar', module)
  .add('default', () => (
    <Avatar src="https://www.justgiving.com/content/images/graphics/icons/avatars/facebook-avatar.gif" />
  ));
