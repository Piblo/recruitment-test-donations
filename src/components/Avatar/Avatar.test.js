import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Avatar from './Avatar';

describe('<Avatar>', () => {
  it('Matches the snapshot', () => {
    const tree = renderer.create(<Avatar src={'url'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
