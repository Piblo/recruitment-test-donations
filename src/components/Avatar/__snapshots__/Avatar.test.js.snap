// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar> Matches the snapshot 1`] = `
.c0 {
  border-radius: 50%;
  height: 100%;
}

<img
  className="c0"
  src="url"
/>
`;
